### PR TITLE
Finalize multi select mobile layout

### PR DESCRIPTION
**Pull request by jDramaix**

### CODE REVIEW

File: plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java [hunk 1]
```diff
@@ -1165,7 +1155,7 @@ private void resultDeselect(int index, String value) {
         searchFieldScale(fWidth);
     }
 
-    private void resultDoHighlight(GQuery el) {
+    protected void resultDoHighlight(GQuery el) {
```
Comment (christiangoudreau) [inline plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java]: still 1000 lines of code? :D

**Comment by christiangoudreau** [comment]
LGTM, @Chris-V and @meriouma could you review?

File: plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java (hunk 1 above)
Comment (jDramaix) [inline plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java]: yeah it's a complex component :)

Comment (christiangoudreau) [inline plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java]: I know, but is there any responsibilities that we could delegate to a new class and reduce the complexity of this one? I should read the complete class, but it is really rare that you can't find anything to extract on a composite so big and complex :D

Comment (jDramaix) [inline plugin/src/main/java/com/arcbees/chosen/client/ChosenImpl.java]: you could still gather code of each feature in a specific class and "inject" an instance of these classes in the component and the component could delegate some logic to these classes. But that will still need days of development.

I think we have already reduce the complexity of the component by removing a lot of if/elseif/else blocks.

**Comment by jDramaix** [comment]
One of the tests failed with TeamCity but work locally... Someone could try to launch the tests on a wiindows machine ?

**Comment by christiangoudreau** [comment]
Relunch just to make sure

**Comment by jDramaix** [comment]
I will merge in order to simplify merge for Jason

**Pull request merged**